website/pages/chapters/ridge.js

```diff
@@ -0,0 +1,845 @@
+              <IM math={`RSS(w) + \\lambda||w||_2^2`} /> will also tend towards{" "}
+              <IM math={`\\infty`} />. We use the intuition that any non-zero
+              number times a number that goes to infinity, will also go to
+              infinity. This means{" "}
+              <IM math={`\\lambda||x||_2^2 \\rightarrow \\infty`} /> since{" "}
+              <IM math={`||x||_2^2 > 0`} />. Even if that magnitude is small,
+              you can blow up that penalty by choosing a super large{" "}
+              <IM math={`\\lambda`} />.
+            </p>
+
+            <p>
+              Consider a different case where every coefficient{" "}
+              <IM math={`j`} />, <IM math={`w_j = 0`} />. We would say then that
+              the whole vector <IM math={`w = 0`} /> as a notational
+              convenience. Then <IM math={`||w||_2^2 = 0`} /> since all of its
+              coefficients are 0. We can then argue that{" "}
+              <IM math={`\\lambda ||w||_2^2 = 0`} /> since any number times 0 is
+              0. So in the case when <IM math={`w = 0`} />, our quality metric
+              is just <IM math={`RSS(0)`} />.
+            </p>
+
+            <p>
+              While the training error for the predictor of all coefficients of
+              0 will likely be high, it is surely less than{" "}
+              <IM math={`\\infty`} />. Therefore, the coefficients that minimize
+              this quality metric will be <IM math={`\\hat{w} = 0`} />.
+            </p>
+          </TYU.Explanation>
+        </TYU>
+
+        <p>
+          So now we have seen both extremes. If you set{" "}
+          <IM math={`\\lambda = 0`} /> you are essentially not caring about
+          regularization, and will favor the least-squares solution. If you make{" "}
+          <IM math={`\\lambda`} /> too large, you will penalize models that have
+          any non-zero coefficients since the cost from regularization will
+          outweigh the loss function. If you set it to something in-between that
+          is more reasonable, you will expect the coefficient magnitudes to be
+          between those extremes as well.
+        </p>
+        <p>
+          In the following graph, you can see an example of how the magnitude of
+          the coefficients for a housing model decrease as you increase{" "}
+          <IM math={`\\lambda`} />. Like in our discussion before, you can
+          imagine that if you let <IM math={`\\lambda \\rightarrow \\infty`} />,
+          all of the coefficients will be 0. Each vertical-line of circles
+          represents a single ridge regression predictor (trained with that{" "}
+          <IM math={`\\lambda`} />
+          ). Each feature's coefficient is associated a color, and you can see
+          how that coefficient changes as you increase <IM math={`\\lambda`} />.
+        </p>
+
+        <figure className="fullwidth">
```

Comment:
seems weird that the image bleeds into the footnotes column, but this is just a personal preference.

Reply:
I could go back to it being in the main column. The Tufte style allows these fullwidth images to provide more detail. I kind of like how it looks, but if it bothers you I can change it.

Reply:
I think for personal preference comments, i'll just leave them when i see something i think should change, and you should just be the final decision :P. If i feel very strongly about something I won't tag is as a personal preference.